backend/src/main.rs

```diff
@@ -39,7 +47,12 @@ fn rocket() -> _ {
 
     rocket::build()
         .attach(FlukeDb::init())
-        .attach(AdHoc::try_on_ignite("SQLx Migrations", run_migrations))
-        .mount("/", routes![index])
+        .attach(fairing::AdHoc::try_on_ignite(
+            "SQLx Migrations",
+            run_migrations,
+        ))
+        .mount("/", routes![all_options])
         .attach(messages::messages_stage())
+        .attach(users::users_stage())
+        .register("/", catchers![not_found])
```

Comment:
```suggestion
        .register("/", catchers![not_found])
        .mount("/", routes![all_options])
        .attach(messages::messages_stage())
        .attach(users::users_stage())
```
nit, The catch all is a root level thing, just a little nicer to have it attached earlier, a ll of our stuff can come after.